player/command.c

```diff
@@ -2137,6 +2137,10 @@ static int get_track_entry(int item, int action, void *arg, void *ctx)
     struct replaygain_data rg = has_rg ? *track->stream->codec->replaygain_data
                                        : (struct replaygain_data){0};
 
+    double par = 0.0;
+    if (p.par_w)
```

Comment:
Shouldn't this be par_h?

Reply:
May need to check both of them. FFmpeg sets numerator to 0 to represent unknown/undefined aspect ratio, and the behavior of mpv built-in demuxer is uncertain.

Reply:
If par_w is 0, the code below will just not expose an aspect sub-property.

Reply:
it makes sense